server/issue.go

```diff
@@ -9,7 +9,6 @@ import (
 	"fmt"
```

Comment:
This entire file will go away once we migrate the issues handler right?

Reply:
Yes, my plan to consolidate handler related code to `xxx_handler.go` in the long run.